std/concurrency.d

```diff
@@ -334,11 +334,22 @@ public:
     void toString(scope void delegate(const(char)[]) sink)
     {
         import std.format;
-        formattedWrite(sink, "Tid(%x)", &mbox);
+        formattedWrite(sink, "Tid(%x)", cast(void*)mbox);
     }
 
 }
 
+unittest
+{
+    import std.conv : text;
+    Tid tid;
+    assert(text(tid) == "Tid(0)");
```

Comment:
Might be better to not depend on the representation and just assert that `text(tid) == text(tid3)`